transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -41,10 +42,13 @@
  * The default {@link ChannelPipeline} implementation.  It is usually created
  * by a {@link Channel} implementation when the {@link Channel} is created.
  */
-final class DefaultChannelPipeline implements ChannelPipeline {
+public class DefaultChannelPipeline implements ChannelPipeline {
```

Comment:
Do we currently have a case where we need to subclass this? I don't see it in this PR.